internal/app/machined/pkg/controllers/runtime/kernel_param_defaults.go

```diff
@@ -95,15 +96,22 @@ func (ctrl *KernelParamDefaultsController) getKernelParams() []*kernel.Param {
 		}...)
 	}
 
+	// Apply IPv6 defaults only if IPv6 is enabled.
+	_, err := os.Stat("/proc/sys/net/ipv6/conf/default/accept_ra")
+	if err == nil {
```

Comment:
I think the `sysctl` binary also silently skips if you try to set non-existent settings in `sysctl.conf` - maybe this should be handled more generically while applying the options, instead of special-casing that specific setting here.

Reply:
Yes, I like that idea better.  Thanks